src/Identity/Extensions.Core/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+Microsoft.AspNetCore.Identity.UserManager<TUser>.RegisterTokenProviderCore(string! providerName, Microsoft.AspNetCore.Identity.IUserTwoFactorTokenProvider<TUser!>! provider) -> void
```

Comment:
We'll probably want an API review issue for this.

Reply:
Updated the linked issue.